### PR TITLE
Add new calc console app

### PR DESCRIPTION
**Pull request by alemyis**
This adds four new math operators. Please review

### CODE REVIEW

**Comment by feridsherif** [comment]
Pull has been reviewed!

File: SimpleHelloWorldTestWithGit/SimpleHelloWorldTestWithGit/Program.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
+        }
+
+        public static int Add(int a, int b)
+        {
+            throw new NotImplementedException("Someone please impliment this");
```
Comment (alemyis) [inline SimpleHelloWorldTestWithGit/SimpleHelloWorldTestWithGit/Program.cs]: This is not complete. Please implement before PR

Comment (alemyis) [reply SimpleHelloWorldTestWithGit/SimpleHelloWorldTestWithGit/Program.cs]: Ok. i will send an update.

**Comment by alemyis** [comment]
First develop to master branch sync.

**Pull request merged**